src/views/Networking.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <Page class="league-page-container">
+    <div
+      class="league-page"
+    >
+      <div id="try_that"></div>
+    </div>
+  </Page>
+</template>
+<script>
+import { RpcAepp, Node } from '@aeternity/aepp-sdk/es';
+import Detector from '@aeternity/aepp-sdk/es/utils/aepp-wallet-communication/wallet-detector';
+import BrowserWindowMessageConnection from '@aeternity/aepp-sdk/es/utils/aepp-wallet-communication/connection/browser-window-message';
+import Page from '../components/layout/Page.vue';
+
+const nodeUrl = 'https://mainnet.aeternity.io';
+
+export default {
+  name: 'League',
+  components: {
+    Page,
+  },
+  data: () => ({ sdk: null }),
+
+  async created() {
+    const sdk = await RpcAepp({
+      name: 'JWT token generator',
+      compilerUrl: 'https://compiler.aepps.com',
+      nodes: [{
+        name: 'why should I have node defined if I\'m not using it?',
+        instance: await Node({ url: nodeUrl, internalUrl: nodeUrl }),
+      }],
+    });
+    const scannerConnection = await BrowserWindowMessageConnection({
+      connectionInfo: { id: 'spy' },
+    });
+    const detector = await Detector({ connection: scannerConnection });
+    await new Promise((resolve) => {
+      detector.scan(async ({ wallets, newWallet }) => {
+        const wallet = newWallet || Object.values(wallets)[0];
+        console.log(`Connecting to wallet ${wallet.name}`);
+        detector.stopScan();
+        await sdk.connectToWallet(await wallet.getConnection());
+        resolve();
+      });
+    });
+    await sdk.subscribeAddress('subscribe', 'current');
+    this.sdk = sdk;
```

Comment:
I think this is not needed. The superhero-ui has wallet detector implemented already.

Reply:
@mradkov thanks fot notice.
how can I be sure that I already have inited wallet if I directly open page like `conference/test` not with router, I mean if someone share it with me. It looks like I need to make my page know that it's inited.

Reply:
> @mradkov thanks fot notice.
> how can I be sure that I already have inited wallet if I directly open page like `conference/test` not with router, I mean if someone share it with me. It looks like I need to make my page know that it's inited.

The wallet detector is invoked when you open the page so if you have the extension installed it will do its job and detect your wallet when you open the page.

Now when I'm thinking more about it - the problem might arise when you don't have the extension installed (in that case we use the web-wallet and deeplinks). You can request signature via the web-wallet (wallet.superhero.com) and authenticate the user that way.

Reply:
Still didn't figured out with situation when I open a page directly and I have an extension, with router it's works fine and `wallet` is not undefinded. And wallet inits async and I just don't know when, but I thought that I's preinited as you said.
Thanks I will try to make it work without extension too.

Reply:
Also mobile support should be tested and implemented.

Reply:
the problem is because `wallet.client` is null in some reason... hope I will figure out next week

Reply:
Also I figured out how to do that with `EventBus` but I don't like that solution. I just defined `EventBus.$emit('clientReady');` after the `this.client = await RpcAepp({})`  at `src/utils/walletSearch.js` and next use `EventBus.$on('clientReady', () => {})`
But it looks like something wrong. Still don't get why `this.client` is not ready just by `init`

Reply:
@mradkov cool, I see that we have other implemetation for that https://github.com/aeternity/superhero-ui/blob/d000ae24a2b8f1b87beb398aed54017b0c24d06d/src/utils/aeternity.js#L31
maybe that will help next week

Reply:
@mradkov fixed with 
```js
EventBus.$emit('clientLive', client);
```

Reply:
currently with 
https://github.com/aeternity/superhero-ui/pull/618/commits/762e2434f09c834a628f1393972ef011bfc2618c#diff-6472c92dade728d085bbecb69253f735R25

I will add support without SDK soon